test/rosdep_duplicates_test.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+
+import os
+
+from scripts.check_duplicates import main as check_duplicates
+
+from .fold_block import Fold
+
+
+def test():
+    files = os.listdir('rosdep')
+    files = filter(lambda x: x.endswith('.yaml'), files) # accept only file ends with .yaml
```

Comment:
Please use a list comprehension for this instead. Also please follow PEP 8 for the comments (two spaces + `#` + one space + comment text)